specificatie/genereervariant/openapi.json

```diff
@@ -4353,6 +4353,13 @@
               "$ref" : "#/components/schemas/ZakelijkGerechtigdeHal"
             }
           },
+          "andereErfpachters" : {
+            "type" : "array",
+            "description" : "De eigenaren van de appartementsrechten die uit erfpacht zijn ontstaan\n",
```

Comment:
\n?

Reply:
Dit is JSON-notatie. Achter elke description staat er een \n. Hij val nu op omdat je de diff ziet,maar eigenlijk kan je het beste alleen de openapi.yaml reviewen.